assets/pages/contributions-landing/components/contributionsBundleContent.jsx

```diff
@@ -0,0 +1,52 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+import { connect } from 'react-redux';
+
+import ContributionsIntroduction from './contributionsIntroduction';
+import ContributionsContext from './contributionsContext';
+import ContributionsContextIntro from './contributionsContextIntro';
+import ContributionsBundle from './contributionsBundle';
+
+
+// ----- Types ----- //
+
+type PropTypes = {
+  context: ?boolean,
```

Comment:
I would set up as a type `boolean`. Since the absence of context is false.

Reply:
Done.